torchvision/models/maxvit.py

```diff
@@ -0,0 +1,595 @@
+import math
+from typing import Any, Callable, List, OrderedDict, Sequence, Tuple
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+from torch import nn, Tensor
+from torchvision.ops.misc import Conv2dNormActivation, SqueezeExcitation
+from torchvision.ops.stochastic_depth import StochasticDepth
+
+
+def get_relative_position_index(height: int, width: int) -> torch.Tensor:
+    coords = torch.stack(torch.meshgrid([torch.arange(height), torch.arange(width)]))
+    coords_flat = torch.flatten(coords, 1)
+    relative_coords = coords_flat[:, :, None] - coords_flat[:, None, :]
+    relative_coords = relative_coords.permute(1, 2, 0).contiguous()
+    relative_coords[:, :, 0] += height - 1
+    relative_coords[:, :, 1] += width - 1
+    relative_coords[:, :, 0] *= 2 * width - 1
+    return relative_coords.sum(-1)
+
+
+class GeluWrapper(nn.Module):
```

Comment:
This wrapper isn't necessary. You can pass `inplace=None` to `ConvNormActivation2D`. See:
https://github.com/pytorch/vision/blob/0a919dbb825d3f33eefcaaaafd3a2bbf6ac91fb8/torchvision/models/swin_transformer.py#L294